pkg/interface/src/views/landscape/components/NewGroup.tsx

```diff
@@ -16,7 +16,9 @@ import useMetadataState from '~/logic/state/metadata';
 import { AsyncButton } from '~/views/components/AsyncButton';
 
 const formSchema = Yup.object({
-  title: Yup.string().required('Group must have a name'),
+  title: Yup.string()
+    .matches(/^(\D).+$/, 'Group names may not start with a number')
```

Comment:
The `+` should really be a `*` I think — single character group names should be allowed, and not error confusingly.

<img width="296" alt="image" src="https://user-images.githubusercontent.com/20846414/121942823-863c3780-cd16-11eb-939a-6fe0d583d799.png">